spring-cloud-aws-messaging/pom.xml

```diff
@@ -55,6 +55,11 @@
 			<artifactId>spring-webmvc</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
```

Comment:
Please remove this dependency and reimplement.

Reply:
Sure. Take a look now. Also added support for non-qualified wrapper types (e.g., `Number.Integer` works like `Number.int`—no need for `Number.java.lang.Integer`).

Reply:
@spencergibb Any comments?